tensorflow/core/util/mkl_util.h

```diff
@@ -2162,6 +2162,12 @@ void execute_primitives(
 }
 #endif  // ENABLE_MKLDNN_V1
 
+#ifdef ENABLE_MKLDNN_V1
+#define GET_USR_MEM_PRIM_DESC(src) src.GetUsrMemDesc()
+#else
+#define GET_USR_MEM_PRIM_DESC(src) src.GetUsrMemPrimDesc()
+#endif  // ENABLE_MKLDNN_V1
+
```

Comment:
Please define these in `mkl_types.h` instead. Please also revert this file since the formatting changes seems inconsistent with our formatter. (If you are using an older version of `clang-format`, please update it to version 8.0.1.)

Reply:
Thank you for the review comments. Sure will do